wallets/cosmos-extension-metamask/src/extension/registry.ts

```diff
@@ -0,0 +1,29 @@
+import { Wallet } from '@cosmos-kit/core';
+
+import { ICON } from '../constant';
+
+export const cosmosSnapExtensionInfo: Wallet = {
+  name: 'cosmos-extension-metamask',
+  description:
+    'The Official Cosmos Extension for MetaMask allowing full use of MetaMask within the Cosmos.',
+  prettyName: 'Cosmos MetaMask Extension',
+  logo: ICON,
+  mode: 'extension',
+  metamask_snap: true,
```

Comment:
@schnetzlerjoe  Hi Joe, can you please replace `metamask_snap: true` to `extends: 'MetaMask'`. We have changed it in Leap's Snap.

Reply:
@twhy Done! :)